c_binding/libsg.c

```diff
@@ -803,3 +813,130 @@ int _sg_tp_sas_addr_of_disk(char *err_msg, int fd, char *tp_sas_addr)
 
     return rc;
 }
+
+int _sg_io_mode_sense(char *err_msg, int fd, uint8_t page_code,
```

Comment:
In SCSI SPC-5 rev12:

    NOTE 20 - Implementations should migrate from the MODE SENSE(6) command to the
    MODE SENSE(10) command.

Hence I intend to no use mode sense 6 any more.